# Georgia and Florida Reopening Among COVID-19 Concerns

## Applied Data Science Capstone by IBM/Coursera

## Introduction/Business Problem
There are cities like Jacksonville reopening beaches while Georgia is planning to reopen several businesses starting April 24th. Is this a good time to consider reopening with the risk to public health from  COVID-19 or will there be too much economic damage if not? In order to answer the questions, I will take a look at county health population information from https://www.countyhealthrankings.org/ and business information(leveraging Foursquare API)in Jacksonsville, Florida and compare to Savannah, Georgia since they are large cities located in the neighboring states. The governers of Georgia and Florida should take these factors into consideration before opening their states among COVID-19.

## Data
<b>I. Neighborhood Locations</b> <br>
I will be using http://www.city-data.com to get neighborhoods and geographical locations. 

<b>II. Foursquare Location API</b> <br>
I will explore business information such as number and type of businesses to see the type of impact closing the cities will have. I also want to see if there are nearby clinics, hospitals, and/or pharmacies to learn about capacity for handling rise in cases for decision in reopening. 

<b>III. County Health Rankings</b> <br>
I will be using county health rankings to get details about the population using information such as:
- Quality of Life
- Health Behaviors
- Clinical Care
- Social & Economic Factors

<b>IV. Death Rates by Age</b> <br>
I will be using https://www.worldometers.info/coronavirus/coronavirus-age-sex-demographics/ to get 
death rates by age. 

<b>V. Hospital Capacities</b> <br>
I will be using https://coronavirus.jhu.edu/us-map to get deaths, confirmed cases, and hospital capacity by county. 

## Methodology

<b>I. Access Foursquare API</b><br>
In order to determine number and/or type of businesses affected, we need to explore Jacksonville and Savannah counties. We will access the Foursquare API to get business information 
but first need to find all the nearby neighborhoods with geographical coordinates in each city. 

Import necessary packages and libraries. 

In [ ]:
import numpy as np # library to handle data in a vectorized manner

import pandas as pd # library for data analsysis
pd.set_option('display.max_columns', None)
pd.set_option('display.max_rows', None)

import json # library to handle JSON files

!conda install -c conda-forge geopy --yes # uncomment this line if you haven't completed the Foursquare API lab
from geopy.geocoders import Nominatim # convert an address into latitude and longitude values

import requests # library to handle requests
from pandas.io.json import json_normalize # tranform JSON file into a pandas dataframe

# Matplotlib and associated plotting modules
import matplotlib.cm as cm
import matplotlib.colors as colors

# import k-means from clustering stage
from sklearn.cluster import KMeans

print('Libraries imported.')

In [2]:
import requests
from bs4 import BeautifulSoup
from requests import get
from requests.exceptions import RequestException
from contextlib import closing
import re

**Generate credentials to access API.**

In [3]:
CLIENT_ID = 'KTOGVYYQUEWMCLZYN3EHY3W1FJ2EUDJ1FIXZKSXQCWBXNQJA' # your Foursquare ID
CLIENT_SECRET = '01RRALCQSNM3UZKYRWZYBNMJL5O5TBFCCBRVKYI4XUA0MSWW' # your Foursquare Secret
VERSION = '20180604'
LIMIT = 30
print('Your credentails:')
print('CLIENT_ID: ' + CLIENT_ID)
print('CLIENT_SECRET:' + CLIENT_SECRET)

Your credentails:
CLIENT_ID: KTOGVYYQUEWMCLZYN3EHY3W1FJ2EUDJ1FIXZKSXQCWBXNQJA
CLIENT_SECRET:01RRALCQSNM3UZKYRWZYBNMJL5O5TBFCCBRVKYI4XUA0MSWW


**Create function for getting neighborhood coordinates by city. The function webscrapes city-data.com to get neighborhood information by city.**

In [4]:
def get_neigborhood(URL,city):
#     URL = 'http://www.city-data.com/nbmaps/neigh-Savannah-Georgia.html'
    page = requests.get(URL)
    soup = BeautifulSoup(page.content, 'html.parser') 
    city_name = [tag.text for tag in soup.find_all('a',href=re.compile(r'#N\d+'))] 
    string=(','+ city)
    city_names=[x + string for x in city_name]
    column_names = ['Neighborhood', 'Latitude', 'Longitude'] 
    # instantiate the dataframe
    neighborhoods = pd.DataFrame(columns=column_names)
    #apply Nominatim
    geolocator = Nominatim(user_agent="to_explorer")
 
    for i in city_names:
        try:
            location = geolocator.geocode(i)
            latitude = location.latitude 
            longitude = location.longitude 
            neighborhoods = neighborhoods.append({'Neighborhood': i,
                                          'Latitude': latitude,
                                          'Longitude': longitude}, ignore_index=True)  
        except AttributeError:
            continue 

    # making separate first name column from new data frame 
    new = neighborhoods["Neighborhood"].str.split(",", n = 1, expand = True) 
    neighborhoods["Neighborhood"]= new[0] 
  
    # making separate last name column from new data frame 
    neighborhoods["City"]= new[1] 
    neighborhoods2 = neighborhoods[["City", "Neighborhood", "Latitude","Longitude"]]


    return neighborhoods2

**Call function by city to get all neighborhoods, latitude,and longitude.**<br>
Let's start with Savannah. 

In [5]:
Jacksonville='http://www.city-data.com/nbmaps/neigh-Jacksonville-Florida.html'
Savannah='http://www.city-data.com/nbmaps/neigh-Savannah-Georgia.html'
neighborhoods=get_neigborhood(Jacksonville,city='Jacksonville')
#neighborhoods=get_neigborhood(Savannah,city='Savannah')
# 'http://www.city-data.com/nbmaps/neigh-Jacksonville-Florida.html',city='Jacksonville'


In [7]:
neighborhoods

City               Neighborhood   Latitude   Longitude
0   Savannah                     Avalon  32.050010  -81.066843
1   Savannah                   Avondale  32.052988  -81.057612
2   Savannah               Baldwin Park  32.050870  -81.091097
3   Savannah                     Bartow  32.086596  -81.124556
4   Savannah                    Bayview  30.224534  -85.682356
5   Savannah            Beach Institute  32.068951  -81.089268
6   Savannah            Berkshire Woods  31.961601  -81.153999
7   Savannah                  Bingville  32.046542  -81.109500
8   Savannah             Blueberry Hill  32.344312  -80.886251
9   Savannah                  Brookview  32.004379  -81.074279
10  Savannah                  Cann Park  32.053430  -81.113216
11  Savannah             Carver Heights  32.060826  -81.099685
12  Savannah           Chatham Crescent  32.050783  -81.096504
13  Savannah            Chatham Parkway  32.062359  -81.160074
14  Savannah                   Chippewa  31.998822  -81.116223
15  Savannah                 Cloverdale  32.066874  -81.129001
16  Savannah               Coffee Bluff  31.936879  -81.153165
17  Savannah              Colonial Oaks  31.962156  -81.148444
18  Savannah           Colonial Village  31.962156  -81.148444
19  Savannah               Dawes Avenue  32.055294  -81.126188
20  Savannah                 Dixon Park  32.060809  -81.090965
21  Savannah          Downtown Savannah  32.075275  -81.098580
22  Savannah              East Savannah  39.941368  -94.829946
23  Savannah                   Eastside  32.046452  -81.066779
24  Savannah                   Edgemere  32.039377  -81.087612
25  Savannah                  Fairfield  32.032710  -81.090667
26  Savannah               Fairway Oaks  32.019933  -81.095945
27  Savannah                   Fernwood  32.032155  -81.064279
28  Savannah                Forest Park  32.080926  -81.091177
29  Savannah               Forest Ridge  32.080926  -81.091177
30  Savannah            Habersham Woods  32.019655  -81.107056
31  Savannah              Highland Park  32.009656  -81.100390
32  Savannah                  Hillcrest  37.727200  -84.269892
33  Savannah              Hitch Village  32.073000  -81.094000
34  Savannah               Jackson Park  35.309266  -83.259593
35  Savannah                     Kayton  32.069785  -81.104518
36  Savannah            Kensington Park  32.023544  -81.101779
37  Savannah               Laurel Grove  32.063056  -81.111917
38  Savannah                   Live Oak  32.083880  -81.195388
39  Savannah                    Mayfair  31.992990  -81.094278
40  Savannah               Medical Arts  32.031369  -81.094137
41  Savannah               Metropolitan -33.451491  -70.731706
42  Savannah                    Oakdale  32.017989  -81.100390
43  Savannah                   Oakhurst  31.995489  -81.124001
44  Savannah               Oakland Park  32.034933  -81.075390
45  Savannah                 Oglethorpe  32.076763  -81.096973
46  Savannah              Paradise Park  33.713487 -117.766140
47  Savannah                   Parkside  32.274694  -80.926254
48  Savannah               Pine Gardens  32.066320  -81.060112
49  Savannah               Poplar Place  35.238418  -88.244227
50  Savannah                  Ridgewood  32.033543  -81.098445
51  Savannah                 Rivers End  31.986878  -81.115389
52  Savannah                   Rose Dhu  31.927992  -81.133720
53  Savannah           Savannah Gardens  32.061321  -81.060945
54  Savannah  Savannah State University  32.022924  -81.056107
55  Savannah               Shirley Park  32.024100  -81.079279
56  Savannah               South Garden  32.080926  -81.091177
57  Savannah                  Southover  32.046875  -81.127056
58  Savannah                Sunset Park  32.034933  -81.065945
59  Savannah             Sylvan Terrace  32.019657  -81.042890
60  Savannah              Thomas Square  31.816875  -81.358855
61  Savannah               Tremont Park  32.059930  -81.137890
62  Savannah            Victory Heights

**Next Jacksonville neighborhoods and coordinates.**

In [6]:
neighborhoods.head()

City     Neighborhood   Latitude  Longitude
0  Jacksonville        Arlington  30.335795 -81.602871
1  Jacksonville         Avondale  30.304406 -81.711209
2  Jacksonville  Belfort Station  30.251119 -81.584455
3  Jacksonville         Brooklyn  30.323573 -81.674541
4  Jacksonville  Church District  30.280525 -81.490998

**Use function to get nearby venues.** 

In [7]:
def getNearbyVenues(names, latitudes, longitudes, radius=1000):
    
    venues_list=[]
    for name, lat, lng in zip(names, latitudes, longitudes):
        print(name)
            
        # create the API request URL
        url = 'https://api.foursquare.com/v2/venues/explore?&client_id={}&client_secret={}&v={}&ll={},{}&radius={}&limit={}'.format(
            CLIENT_ID, 
            CLIENT_SECRET, 
            VERSION, 
            lat, 
            lng, 
            radius, 
            LIMIT)
            
        # make the GET request
        results = requests.get(url).json()["response"]['groups'][0]['items']
        
        # return only relevant information for each nearby venue
        venues_list.append([(
            name, 
            lat, 
            lng, 
            v['venue']['name'], 
            v['venue']['location']['lat'], 
            v['venue']['location']['lng'],  
            v['venue']['categories'][0]['name']) for v in results])

    nearby_venues = pd.DataFrame([item for venue_list in venues_list for item in venue_list])
    nearby_venues.columns = ['Neighborhood', 
                  'Neighborhood Latitude', 
                  'Neighborhood Longitude', 
                  'Venue', 
                  'Venue Latitude', 
                  'Venue Longitude', 
                  'Venue Category']
    
    return(nearby_venues)

**Create simple variable to assign different cities.**

In [9]:
A='Savannah'
A

'Savannah'

In [10]:
A_venues = getNearbyVenues(names=neighborhoods['Neighborhood'],
                                   latitudes=neighborhoods['Latitude'],
                                   longitudes=neighborhoods['Longitude']
                                  )


Avalon
Avondale
Baldwin Park
Bartow
Bayview
Beach Institute
Berkshire Woods
Bingville
Blueberry Hill
Brookview
Cann Park
Carver Heights
Chatham Crescent
Chatham Parkway
Chippewa
Cloverdale
Coffee Bluff
Colonial Oaks
Colonial Village
Dawes Avenue
Dixon Park
Downtown Savannah
East Savannah
Eastside
Edgemere
Fairfield
Fairway Oaks
Fernwood
Forest Park
Forest Ridge
Habersham Woods
Highland Park
Hillcrest
Hitch Village
Jackson Park
Kayton
Kensington Park
Laurel Grove
Live Oak
Mayfair
Medical Arts
Metropolitan
Oakdale
Oakhurst
Oakland Park
Oglethorpe
Paradise Park
Parkside
Pine Gardens
Poplar Place
Ridgewood
Rivers End
Rose Dhu
Savannah Gardens
Savannah State University
Shirley Park
South Garden
Southover
Sunset Park
Sylvan Terrace
Thomas Square
Tremont Park
Victory Heights
Victory Manor
Victory Square
West Savannah
Windsor Forest
Winter Gardens
Yamacraw Village


In [11]:
A='Jacksonville'
A_venues = getNearbyVenues(names=neighborhoods['Neighborhood'],
                                   latitudes=neighborhoods['Latitude'],
                                   longitudes=neighborhoods['Longitude']
                                  )

Arlington
Avondale
Belfort Station
Brooklyn
Church District
Downtown
East Jacksonville
Empire Point
Mandarin
North Jacksonville
Oakland
Ortega
River Park
Riverside
San Marco
Sherwood Forest
South Jacksonville
South Side
Springfield
Stadium District
Waterleaf
West Jacksonville


In [11]:
A_venues.groupby('Neighborhood').count()
print('There are {} uniques categories.'.format(len(A_venues['Venue Category'].unique())))

There are 193 uniques categories.


In [12]:
A_venues.groupby('Neighborhood').count()
print('There are {} uniques categories.'.format(len(A_venues['Venue Category'].unique())))

There are 137 uniques categories.


**One-hot Encoding for Generating Venue Frequency**

In [13]:
# one hot encoding
A_onehot = pd.get_dummies(A_venues[['Venue Category']], prefix="", prefix_sep="")

# add neighborhood column back to dataframe
A_onehot['Neighborhood'] = A_venues['Neighborhood'] 

# move neighborhood column to the first column
fixed_columns = [A_onehot.columns[-1]] + list(A_onehot.columns[:-1])
A_onehot = A_onehot[fixed_columns]

A_onehot.head()

Yoga Studio  ATM  Accessories Store  Advertising Agency  \
0            0    0                  0                   0   
1            0    0                  0                   0   
2            0    0                  0                   0   
3            0    0                  0                   0   
4            0    0                  0                   0   

   American Restaurant  Antique Shop  Arcade  Art Gallery  \
0                    0             0       0            0   
1                    0             0       0            0   
2                    0             0       0            0   
3                    0             0       0            0   
4                    0             0       0            0   

   Arts & Crafts Store  Asian Restaurant  Australian Restaurant  \
0                    0                 0                      0   
1                    0                 0                      0   
2                    0                 0                      0   
3                    0                 0                      0   
4                    0                 0                      0   

   Automotive Shop  BBQ Joint  Bagel Shop  Bakery  Bank  Bar  Baseball Field  \
0                0          0           0       0     0    0               0   
1                0          0           0       0     0    0               0   
2                0          0           0       0     0    0               0   
3                0          0           0       0     0    0               0   
4                0          0           0       0     0    0               0   

   Baseball Stadium  Bed & Breakfast  Beer Store  Big Box Store  \
0                 0                0           0              0   
1                 0                0           0              0   
2                 0                0           0              0   
3                 0                0           0              0   
4                 0                0           0              0   

   Bike Rental / Bike Share  Bistro  Boat or Ferry  Bookstore  Boutique  \
0                         0       0              0          0         0   
1                         0       0              0          0         0   
2                         0       0              0          0         0   
3                         0       0              0          0         0   
4                         0       0              0          0         0   

   Bowling Alley  Breakfast Spot  Brewery  Bubble Tea Shop  Building  \
0              0               0        0                0         0   
1              0               0        0                0         0   
2              0               0        0                0         0   
3              0               0        0                0         0   
4              0               0        0                0         0   

   Burger Joint  Burrito Place  Bus Station  Business Service  Café  \
0             0              0            0                 0     0   
1             0              0            0                 0     0   
2             0              0            0                 0     0   
3             0              0            0                 0     0   
4             0              0            0                 0     0   

   Campground  Candy Store  Caribbean Restaurant  Chinese Restaurant  \
0           0            0                     0                   0   
1           0            0                     0                   0   
2           0            0                     0                   1   
3           0            0                     0                   0   
4           0            0                     0                   0   

   Clothing Store  Cocktail Bar  Coffee Shop  College Arts Building  \
0               0             0            0                      0   
1               0             0            0                      0   
2               0  

In [13]:
# one hot encoding
A_onehot = pd.get_dummies(A_venues[['Venue Category']], prefix="", prefix_sep="")

# add neighborhood column back to dataframe
A_onehot['Neighborhood'] = A_venues['Neighborhood'] 

# move neighborhood column to the first column
fixed_columns = [A_onehot.columns[-1]] + list(A_onehot.columns[:-1])
A_onehot = A_onehot[fixed_columns]

A_onehot.head()

Neighborhood  ATM  American Restaurant  Arcade  Art Gallery  Art Museum  \
0    Arlington    0                    0       0            0           0   
1    Arlington    0                    0       0            0           0   
2    Arlington    0                    0       0            0           0   
3    Arlington    0                    0       0            0           0   
4    Arlington    0                    0       0            0           0   

   Arts & Crafts Store  Arts & Entertainment  Asian Restaurant  \
0                    0                     0                 0   
1                    0                     0                 0   
2                    0                     0                 0   
3                    0                     0                 0   
4                    0                     0                 0   

   Automotive Shop  Bagel Shop  Bakery  Bank  Bar  Baseball Field  \
0                0           0       0     0    0               0   
1                0           0       0     0    0               0   
2                0           0       0     0    0               0   
3                0           0       0     0    0               0   
4                0           0       0     0    0               0   

   Baseball Stadium  Beer Garden  Bookstore  Breakfast Spot  Brewery  \
0                 0            0          0               0        0   
1                 0            0          0               0        0   
2                 0            0          0               0        0   
3                 0            0          0               0        0   
4                 0            0          0               0        0   

   Burger Joint  Burrito Place  Business Service  Cafeteria  Café  \
0             0              0                 0          0     0   
1             0              0                 0          0     0   
2             0              0                 0          0     0   
3             0              0                 0          0     0   
4             0              0                 0          0     0   

   Caribbean Restaurant  Chinese Restaurant  Chiropractor  Cocktail Bar  \
0                     0                   0             0             0   
1                     0                   0             0             0   
2                     0                   0             0             0   
3                     0                   0             0             0   
4                     0                   0             0             0   

   Coffee Shop  Comfort Food Restaurant  Concert Hall  Convenience Store  \
0            0                        0             0                  0   
1            0                        0             0                  0   
2            0                        0             0                  0   
3            0                        0             0                  0   
4            0                        0             0                  0   

   Cosmetics Shop  Deli / Bodega  Dentist's Office  Dessert Shop  Diner  \
0               0              0                 0             0      0   
1               0              0                 0             0      0   
2               0              0                 0             0      0   
3               0              0                 0             0      0   
4               0              0                 0             0      0   

   Disc Golf  Discount Store  Dive Bar  Dog Run  Donut Shop  Event Service  \
0          0               0         0        0           1              0   
1          0               0         0        0           0              0   
2          0               0         0        0           0              0   
3          0               1         0        0           0              0   
4          0               0         0        0           0              0   

   Exhibit  Farmers Market  Fast Food Restaurant  F

In [14]:
#Savannah
A_grouped = A_onehot.groupby('Neighborhood').mean().reset_index()
A_grouped

Neighborhood  Yoga Studio       ATM  Accessories Store  \
0                      Avalon     0.000000  0.000000           0.000000   
1                    Avondale     0.000000  0.000000           0.000000   
2                Baldwin Park     0.000000  0.000000           0.000000   
3                      Bartow     0.000000  0.000000           0.000000   
4                     Bayview     0.000000  0.000000           0.000000   
5             Beach Institute     0.000000  0.000000           0.000000   
6             Berkshire Woods     0.000000  0.000000           0.000000   
7                   Bingville     0.041667  0.000000           0.000000   
8              Blueberry Hill     0.000000  0.000000           0.000000   
9                   Brookview     0.000000  0.000000           0.000000   
10                  Cann Park     0.000000  0.000000           0.000000   
11             Carver Heights     0.033333  0.000000           0.000000   
12           Chatham Crescent     0.000000  0.000000           0.000000   
13            Chatham Parkway     0.000000  0.000000           0.000000   
14                   Chippewa     0.000000  0.000000           0.033333   
15                 Cloverdale     0.000000  0.000000           0.000000   
16               Coffee Bluff     0.000000  0.000000           0.000000   
17              Colonial Oaks     0.000000  0.000000           0.000000   
18           Colonial Village     0.000000  0.000000           0.000000   
19               Dawes Avenue     0.000000  0.000000           0.000000   
20                 Dixon Park     0.033333  0.000000           0.000000   
21          Downtown Savannah     0.000000  0.000000           0.000000   
22              East Savannah     0.000000  0.000000           0.000000   
23                   Eastside     0.000000  0.000000           0.000000   
24                   Edgemere     0.000000  0.000000           0.000000   
25                  Fairfield     0.000000  0.000000           0.000000   
26               Fairway Oaks     0.000000  0.000000           0.000000   
27                   Fernwood     0.000000  0.000000           0.000000   
28                Forest Park     0.000000  0.000000           0.000000   
29               Forest Ridge     0.000000  0.000000           0.000000   
30            Habersham Woods     0.000000  0.000000           0.000000   
31              Highland Park     0.000000  0.000000           0.000000   
32                  Hillcrest     0.000000  0.000000           0.052632   
33              Hitch Village     0.000000  0.000000           0.000000   
34               Jackson Park     0.000000  0.000000           0.000000   
35                     Kayton     0.000000  0.000000           0.000000   
36            Kensington Park     0.000000  0.000000           0.000000   
37               Laurel Grove     0.000000  0.000000           0.000000   
38                   Live Oak     0.000000  0.000000           0.000000   
39                    Mayfair     0.000000  0.000000           0.000000   
40               Medical Arts     0.033333  0.000000           0.000000   
41               Metropolitan     0.000000  0.000000           0.000000   
42                    Oakdale     0.000000  0.000000           0.000000   
43                   Oakhurst     0.000000  0.000000           0.033333   
44               Oakland Park     0.000000  0.000000           0.000000   
45                 Oglethorpe     0.000000  0.000000           0.000000   
46              Paradise Park     0.000000  0.033333           0.000000   
47                   Parkside     0.000000  0.000000           0.000000   
48               Pine Gardens     0.000000  0.000000           0.000000   
49               Poplar Place     0.000000  0.000000           0.000000   
50                  Ridgewood     0.033333  0.000000           0.000000   
51                 Rivers End     0.000000  0.000000           0.000000   
52                   Rose Dhu     0.000000

In [15]:
#Jacksonville
A_grouped = A_onehot.groupby('Neighborhood').mean().reset_index()
A_grouped.head()

Neighborhood       ATM  American Restaurant    Arcade  Art Gallery  \
0        Arlington  0.041667             0.041667  0.000000          0.0   
1         Avondale  0.000000             0.000000  0.000000          0.0   
2  Belfort Station  0.000000             0.000000  0.058824          0.0   
3         Brooklyn  0.000000             0.000000  0.000000          0.0   
4  Church District  0.000000             0.000000  0.000000          0.0   

   Art Museum  Arts & Crafts Store  Arts & Entertainment  Asian Restaurant  \
0    0.000000             0.000000                   0.0               0.0   
1    0.000000             0.000000                   0.0               0.0   
2    0.000000             0.000000                   0.0               0.0   
3    0.033333             0.033333                   0.0               0.0   
4    0.000000             0.000000                   0.0               0.0   

   Automotive Shop  Bagel Shop    Bakery  Bank       Bar  Baseball Field  \
0         0.041667         0.0  0.000000   0.0  0.041667             0.0   
1         0.000000         0.0  0.041667   0.0  0.041667             0.0   
2         0.058824         0.0  0.000000   0.0  0.000000             0.0   
3         0.000000         0.0  0.000000   0.0  0.000000             0.0   
4         0.000000         0.0  0.000000   0.0  0.000000             0.0   

   Baseball Stadium  Beer Garden  Bookstore  Breakfast Spot   Brewery  \
0               0.0          0.0   0.041667        0.000000  0.000000   
1               0.0          0.0   0.000000        0.041667  0.041667   
2               0.0          0.0   0.000000        0.000000  0.000000   
3               0.0          0.0   0.000000        0.000000  0.000000   
4               0.0          0.0   0.000000        0.000000  0.000000   

   Burger Joint  Burrito Place  Business Service  Cafeteria  Café  \
0      0.000000       0.000000          0.000000        0.0   0.0   
1      0.000000       0.000000          0.041667        0.0   0.0   
2      0.000000       0.000000          0.000000        0.0   0.0   
3      0.033333       0.033333          0.000000        0.0   0.1   
4      0.000000       0.000000          0.000000        0.0   0.0   

   Caribbean Restaurant  Chinese Restaurant  Chiropractor  Cocktail Bar  \
0                   0.0            0.000000           0.0      0.041667   
1                   0.0            0.041667           0.0      0.000000   
2                   0.0            0.058824           0.0      0.058824   
3                   0.0            0.000000           0.0      0.000000   
4                   0.0            0.047619           0.0      0.000000   

   Coffee Shop  Comfort Food Restaurant  Concert Hall  Convenience Store  \
0     0.000000                      0.0      0.000000           0.000000   
1     0.041667                      0.0      0.000000           0.083333   
2     0.000000                      0.0      0.000000           0.000000   
3     0.000000                      0.0      0.033333           0.000000   
4     0.000000                      0.0      0.000000           0.000000   

   Cosmetics Shop  Deli / Bodega  Dentist's Office  Dessert Shop  Diner  \
0             0.0       0.000000          0.000000      0.000000    0.0   
1             0.0       0.041667          0.000000      0.000000    0.0   
2             0.0       0.000000          0.000000      0.000000    0.0   
3             0.0       0.066667          0.000000      0.033333    0.0   
4             0.0       0.000000          0.047619      0.000000    0.0   

   Disc Golf  Discount Store  Dive Bar  Dog Run  Donut Shop  Event Service  \
0        0.0        0.041667  0.000000      0.0    0.041667            0.0   
1        0.0        0.000000  0.000000      0.0    0.000000            0.0   
2        0.0        0.000000  0.000000      0.0    0.000000            0.0   
3        0.0        0.000000  0.033333      0.0    0.000000            0.0   
4        0.0        

### Savannah Venue Frequency

In [20]:
num_top_venues = 10
for hood in A_grouped['Neighborhood']:
    print("----"+hood+"----")
    temp = A_grouped[A_grouped['Neighborhood'] == hood].T.reset_index()
    temp.columns = ['venue','freq']
    temp = temp.iloc[1:]
    temp['freq'] = temp['freq'].astype(float)
    temp = temp.round({'freq': 2})
    print(temp.sort_values('freq', ascending=False).reset_index(drop=True).head(num_top_venues))
    print('\n')
# Double click to see results

----Avalon----
                 venue  freq
0       Sandwich Place  0.07
1          Pizza Place  0.07
2   Mexican Restaurant  0.07
3    Mobile Phone Shop  0.07
4   Chinese Restaurant  0.07
5       Breakfast Spot  0.03
6        Grocery Store  0.03
7     Greek Restaurant  0.03
8  Fried Chicken Joint  0.03
9                 Park  0.03


----Avondale----
                    venue  freq
0                    Park  0.13
1  Furniture / Home Store  0.07
2           Historic Site  0.07
3     Sporting Goods Shop  0.07
4            Skating Rink  0.07
5                    Food  0.07
6          General Travel  0.07
7      Seafood Restaurant  0.07
8    Fast Food Restaurant  0.07
9            Liquor Store  0.07


----Baldwin Park----
                             venue  freq
0                      Gas Station  0.10
1  Southern / Soul Food Restaurant  0.07
2                     Intersection  0.07
3                       Playground  0.07
4              American Restaurant  0.07
5                         

*In East Savannah, pizza places(18%) and discount stores(18%) account for 36% of venues. In Southover, Savannah bakeries(25%), breweries(25%), and discount stores(25%) account for 75% of venues.* 

### Jacksonville Venue Frequency

In [16]:

num_top_venues = 10
for hood in A_grouped['Neighborhood']:
    print("----"+hood+"----")
    temp = A_grouped[A_grouped['Neighborhood'] == hood].T.reset_index()
    temp.columns = ['venue','freq']
    temp = temp.iloc[1:]
    temp['freq'] = temp['freq'].astype(float)
    temp = temp.round({'freq': 2})
    print(temp.sort_values('freq', ascending=False).reset_index(drop=True).head(num_top_venues))
    print('\n')

----Arlington----
                 venue  freq
0           Restaurant  0.08
1          Video Store  0.08
2                  ATM  0.04
3            Bookstore  0.04
4           Donut Shop  0.04
5       Discount Store  0.04
6       Sandwich Place  0.04
7   Seafood Restaurant  0.04
8        Shopping Mall  0.04
9  Fried Chicken Joint  0.04


----Avondale----
               venue  freq
0        Pizza Place  0.08
1  Convenience Store  0.08
2               Park  0.08
3         Smoke Shop  0.04
4      Historic Site  0.04
5  French Restaurant  0.04
6              Plaza  0.04
7        Gaming Cafe  0.04
8      Deli / Bodega  0.04
9        Coffee Shop  0.04


----Belfort Station----
                 venue  freq
0                Hotel  0.18
1       Sandwich Place  0.12
2               Arcade  0.06
3   Chinese Restaurant  0.06
4  Japanese Restaurant  0.06
5          IT Services  0.06
6   Mexican Restaurant  0.06
7                  Gym  0.06
8          Gas Station  0.06
9           Food Truck  0.06




*In North Jacksonville and East Jacksonville, fast food restaurants account for about 38% of venues. In East Savannah, pizza places(18%) and discount stores(18%) account for 36% of venues. In Southover, Savannah bakeries(25%), breweries(25%), and discount stores(25%) account for 75% of venues.* 

**Get most common revenues and place in dataframe.**

In [17]:
def return_most_common_venues(row, num_top_venues):
    row_categories = row.iloc[1:]
    row_categories_sorted = row_categories.sort_values(ascending=False)
    
    return row_categories_sorted.index.values[0:num_top_venues]

**Savannah Top 10 Revenues**

In [17]:
num_top_venues = 10

indicators = ['st', 'nd', 'rd']

# create columns according to number of top venues
columns = ['Neighborhood']
for ind in np.arange(num_top_venues):
    try:
        columns.append('{}{} Most Common Venue'.format(ind+1, indicators[ind]))
    except:
        columns.append('{}th Most Common Venue'.format(ind+1))

# create a new dataframe
A_neighborhoods_venues_sorted = pd.DataFrame(columns=columns)
A_neighborhoods_venues_sorted['Neighborhood'] = A_grouped['Neighborhood']

for ind in np.arange(A_grouped.shape[0]):
    A_neighborhoods_venues_sorted.iloc[ind, 1:] = return_most_common_venues(A_grouped.iloc[ind, :], num_top_venues)

A_neighborhoods_venues_sorted.head(10)

Neighborhood 1st Most Common Venue 2nd Most Common Venue  \
0           Avalon     Mobile Phone Shop           Pizza Place   
1         Avondale                  Park   Sporting Goods Shop   
2     Baldwin Park           Gas Station                  Park   
3           Bartow             Pet Store                  Park   
4          Bayview                Garden            Kids Store   
5  Beach Institute                 Plaza       Bed & Breakfast   
6  Berkshire Woods                  Park          Home Service   
7        Bingville     Convenience Store                  Park   
8   Blueberry Hill                  Farm         Women's Store   
9        Brookview    Chinese Restaurant          Intersection   

        3rd Most Common Venue            4th Most Common Venue  \
0          Chinese Restaurant                   Sandwich Place   
1              General Travel                Convenience Store   
2                Intersection                       Playground   
3              Discount Store                             Food   
4            Drive-in Theater                    Women's Store   
5                        Park  Southern / Soul Food Restaurant   
6          Photography Studio                      Coffee Shop   
7              Discount Store                      Wings Joint   
8                 Wings Joint                       Food Truck   
9  Construction & Landscaping                             Food   

    5th Most Common Venue            6th Most Common Venue  \
0      Mexican Restaurant                       Restaurant   
1           Historic Site               Seafood Restaurant   
2     American Restaurant  Southern / Soul Food Restaurant   
3        Business Service               Seafood Restaurant   
4              Donut Shop                Food & Drink Shop   
5                   Hotel               Italian Restaurant   
6              Donut Shop                             Food   
7           Grocery Store            General Entertainment   
8       Food & Drink Shop                             Food   
9  Furniture / Home Store                      Golf Course   

  7th Most Common Venue   8th Most Common Venue  \
0         Big Box Store          Breakfast Spot   
1          Skating Rink    Fast Food Restaurant   
2    Salon / Barbershop     Sporting Goods Shop   
3    Chinese Restaurant           Grocery Store   
4                  Food             Flower Shop   
5              Dive Bar                  Bakery   
6           Flower Shop             Fish Market   
7  Fast Food Restaurant      Mexican Restaurant   
8           Flower Shop             Fish Market   
9     Food & Drink Shop  Thrift / Vintage Store   

             9th Most Common Venue 10th Most Common Venue  
0             Fast Food Restaurant         Hardware Store  
1                             Food           Liquor Store  
2                     Liquor Store                 Lounge  
3                Mobile Phone Shop    American Restaurant  
4                      Fish Market   Fast Food Restaurant  
5                         Tea Room              BBQ Joint  
6             Fast Food Restaurant         Farmers Market  
7                Electronics Store                    Bar  
8             Fast Food Restaurant         Farmers Market  
9  Southern / Soul Food Restaurant      Electronics Store

**Jacksonville Top 10 Revenues**

In [18]:
#Jacksonville
num_top_venues = 10

indicators = ['st', 'nd', 'rd']

# create columns according to number of top venues
columns = ['Neighborhood']
for ind in np.arange(num_top_venues):
    try:
        columns.append('{}{} Most Common Venue'.format(ind+1, indicators[ind]))
    except:
        columns.append('{}th Most Common Venue'.format(ind+1))

# create a new dataframe
A_neighborhoods_venues_sorted = pd.DataFrame(columns=columns)
A_neighborhoods_venues_sorted['Neighborhood'] = A_grouped['Neighborhood']

for ind in np.arange(A_grouped.shape[0]):
    A_neighborhoods_venues_sorted.iloc[ind, 1:] = return_most_common_venues(A_grouped.iloc[ind, :], num_top_venues)

A_neighborhoods_venues_sorted.head(10)

Neighborhood 1st Most Common Venue 2nd Most Common Venue  \
0           Arlington           Video Store            Restaurant   
1            Avondale                  Park           Pizza Place   
2     Belfort Station                 Hotel        Sandwich Place   
3            Brooklyn                  Café                  Park   
4     Church District         Grocery Store           Pizza Place   
5            Downtown                Bakery        History Museum   
6   East Jacksonville  Fast Food Restaurant           Video Store   
7        Empire Point          Chiropractor                Lawyer   
8            Mandarin         Event Service             Pet Store   
9  North Jacksonville  Fast Food Restaurant           Video Store   

  3rd Most Common Venue 4th Most Common Venue 5th Most Common Venue  \
0                   ATM    Italian Restaurant    Mexican Restaurant   
1     Convenience Store           Gaming Cafe                 Plaza   
2       Automotive Shop          Cocktail Bar    Chinese Restaurant   
3         Deli / Bodega  Gym / Fitness Center                   Gym   
4   Japanese Restaurant          Intersection                   Spa   
5            Truck Stop           Gas Station                   Bar   
6    Light Rail Station         Train Station        Sandwich Place   
7   American Restaurant   Rental Car Location           Gas Station   
8      Business Service        Farmers Market                  Park   
9    Light Rail Station         Train Station        Sandwich Place   

  6th Most Common Venue    7th Most Common Venue 8th Most Common Venue  \
0         Shopping Mall       Seafood Restaurant        Sandwich Place   
1         Deli / Bodega  New American Restaurant           Coffee Shop   
2            Food Truck              Gas Station   Japanese Restaurant   
3         Smoothie Shop           Sandwich Place          Concert Hall   
4         Shopping Mall                 Pharmacy          Liquor Store   
5    Italian Restaurant         Greek Restaurant             Rest Area   
6                Bakery                      Zoo         Event Service   
7        Baseball Field               Donut Shop           Flea Market   
8            Food Truck              Flea Market  Fast Food Restaurant   
9                Bakery                      Zoo         Event Service   

  9th Most Common Venue 10th Most Common Venue  
0   Fried Chicken Joint              Bookstore  
1    Chinese Restaurant             Smoke Shop  
2    Italian Restaurant     Mexican Restaurant  
3                 Plaza             Playground  
4        Shipping Store         Sandwich Place  
5   American Restaurant               Dive Bar  
6           Flea Market         Farmers Market  
7  Fast Food Restaurant         Farmers Market  
8               Exhibit                    Zoo  
9           Flea Market         Farmers Market

**Read CSV Files with Table Information**

In [1]:
pip install xlrd

     |████████████████████████████████| 103 kB 3.1 MB/s eta 0:00:01
Note: you may need to restart the kernel to use updated packages.


In [2]:
pwd

'/home/jovyan/demo'

In [2]:
import pandas as pd
County_Table= pd.read_csv('/home/jovyan/demo/County_Health_Rankings.csv')

**County Health Rankings Table**<br>
Reference:https://www.countyhealthrankings.org/app/georgia/2020/compare/snapshot?counties=12_031%2013_051

In [16]:
County_Table[County_Table.isnull()]=""
County_Table.head()

Unnamed: 0 Duval (DU) , FL  Chatham (CA) , GA 
0             Demographics                                    
1               Population          950,181            289,195
2  % below 18 years of age           22.60%             21.20%
3           % 65 and older           14.00%             15.40%
4     % Non-Hispanic Black           29.50%             40.10%

In [6]:
Death_by_Age_Table= pd.read_csv('/home/jovyan/demo/Death_By_Age_Coronavirus.csv')
#Reference:https://www.worldometers.info/coronavirus/coronavirus-age-sex-demographics/
#As of May 13th

## Results

**Table 1. Savannah Venues**

In [19]:
Savannah_data=A_neighborhoods_venues_sorted
Savannah_data
#Double click for results

Neighborhood 1st Most Common Venue  \
0                      Avalon     Mobile Phone Shop   
1                    Avondale                  Park   
2                Baldwin Park           Gas Station   
3                      Bartow             Pet Store   
4                     Bayview                Garden   
5             Beach Institute                 Plaza   
6             Berkshire Woods                  Park   
7                   Bingville     Convenience Store   
8              Blueberry Hill                  Farm   
9                   Brookview    Chinese Restaurant   
10                  Cann Park                Bakery   
11             Carver Heights           Coffee Shop   
12           Chatham Crescent                  Café   
13            Chatham Parkway        Sandwich Place   
14                   Chippewa        Cosmetics Shop   
15                 Cloverdale                  Park   
16               Coffee Bluff    Photography Studio   
17              Colonial Oaks          Home Service   
18           Colonial Village          Home Service   
19               Dawes Avenue        Discount Store   
20                 Dixon Park           Coffee Shop   
21          Downtown Savannah                 Plaza   
22              East Savannah           Pizza Place   
23                   Eastside                  Park   
24                   Edgemere                  Park   
25                  Fairfield              Pharmacy   
26               Fairway Oaks              Pharmacy   
27                   Fernwood        Discount Store   
28                Forest Park           Candy Store   
29               Forest Ridge           Candy Store   
30            Habersham Woods                 Hotel   
31              Highland Park      Asian Restaurant   
32                  Hillcrest        Discount Store   
33              Hitch Village                 Plaza   
34               Jackson Park             BBQ Joint   
35                     Kayton                 Plaza   
36            Kensington Park                 Hotel   
37               Laurel Grove                  Park   
38                   Live Oak     Convenience Store   
39                    Mayfair                Bakery   
40               Medical Arts        Sandwich Place   
41               Metropolitan           Bus Station   
42                    Oakdale              Pharmacy   
43                   Oakhurst        Sandwich Place   
44               Oakland Park        Discount Store   
45                 Oglethorpe                 Plaza   
46              Paradise Park     Korean Restaurant   
47                   Parkside           Golf Course   
48               Pine Gardens     Convenience Store   
49               Poplar Place        Baseball Field   
50                  Ridgewood   American Restaurant   
51                 Rivers End            Playground   
52                   Rose Dhu            Campground   
53           Savannah Gardens     Convenience Store   
54  Savannah State University       Harbor / Marina   
55               Shirley Park    Chinese Restaurant   
56               South Garden           Candy Store   
57                  Southover               Brewery   
58                Sunset Park            Sports Bar   
59             Sylvan Terrace       Harbor / Marina   
60              Thomas Square          Gourmet Shop   
61               Tremont Park           Gas Station   
62            Victory Heights    Chinese Restaurant   
63              Victory Manor           Pizza Place   
64             Victory Square    Chinese Restaurant   
65              West Savannah           Pizza Place   
66             Windsor Forest           Pizza Place   
67             Winter Gardens                  Pool   
68           Yamacraw Village                Museum   

              2nd Most Common Venue            3rd Most Common Venue  \
0                       Pizza Place               Chinese Restaurant   
1               Sporti

**Table 2. Jacksonville Venues**

In [19]:
Jacksonville_data=A_neighborhoods_venues_sorted
Jacksonville_data

Neighborhood 1st Most Common Venue     2nd Most Common Venue  \
0            Arlington           Video Store                Restaurant   
1             Avondale                  Park               Pizza Place   
2      Belfort Station                 Hotel            Sandwich Place   
3             Brooklyn                  Café                      Park   
4      Church District         Grocery Store               Pizza Place   
5             Downtown                Bakery            History Museum   
6    East Jacksonville  Fast Food Restaurant               Video Store   
7         Empire Point          Chiropractor                    Lawyer   
8             Mandarin         Event Service                 Pet Store   
9   North Jacksonville  Fast Food Restaurant               Video Store   
10             Oakland                   Bar       Rental Car Location   
11              Ortega                  Park           Harbor / Marina   
12          River Park           Coffee Shop               Art Gallery   
13           Riverside   American Restaurant          Sushi Restaurant   
14           San Marco           Coffee Shop       American Restaurant   
15     Sherwood Forest           Music Venue            Discount Store   
16  South Jacksonville                 Hotel  Mediterranean Restaurant   
17          South Side                  Park               Pizza Place   
18         Springfield                   Zoo                   Exhibit   
19    Stadium District               Stadium       American Restaurant   
20           Waterleaf            Food Truck              Intersection   
21   West Jacksonville          Home Service            Cosmetics Shop   

        3rd Most Common Venue 4th Most Common Venue 5th Most Common Venue  \
0                         ATM    Italian Restaurant    Mexican Restaurant   
1           Convenience Store           Gaming Cafe                 Plaza   
2             Automotive Shop          Cocktail Bar    Chinese Restaurant   
3               Deli / Bodega  Gym / Fitness Center                   Gym   
4         Japanese Restaurant          Intersection                   Spa   
5                  Truck Stop           Gas Station                   Bar   
6          Light Rail Station         Train Station        Sandwich Place   
7         American Restaurant   Rental Car Location           Gas Station   
8            Business Service        Farmers Market                  Park   
9          Light Rail Station         Train Station        Sandwich Place   
10                       Park    Italian Restaurant                  Café   
11               Home Service         Jewelry Store              Pharmacy   
12                       Café    Mexican Restaurant            Food Truck   
13             Sandwich Place            Restaurant                   Bar   
14   Mediterranean Restaurant        Sandwich Place               Brewery   
15                        Zoo               Exhibit            Food Truck   
16         Mexican Restaurant           Coffee Shop        Sandwich Place   
17             Baseball Field    Salon / Barbershop    Photography Studio   
18                Flower Shop              Tea Room         Deli / Bodega   
19       Fast Food Restaurant           Pizza Place             Pet Store   
20         Photography Studio             Disc Golf        Discount Store   
21  Middle Eastern Restaurant          Intersection        Sandwich Place   

   6th Most Common Venue    7th Most Common Venue 8th Most Common Venue  \
0          Shopping Mall       Seafood Restaurant        Sandwich Place   
1          Deli / Bodega  New American Restaurant           Coffee Shop   
2             Food Truck              Gas Station   Japanese Restaurant   
3          Smoothie Shop           Sandwich Place          Concert Hall   
4          Shopping Mall                 Pharmacy          Liquor Store   
5     Italian Restaurant         Greek Restaurant             Rest Area   
6         

**Table 3. County Health Rankings**<br>
Reference: https://www.countyhealthrankings.org/app/georgia/2020/compare/snapshot?counties=12_031%2013_051 

In [17]:
County_Table

Unnamed: 0 Duval (DU) , FL   \
0                               Demographics                    
1                                 Population          950,181   
2                    % below 18 years of age           22.60%   
3                             % 65 and older           14.00%   
4                       % Non-Hispanic Black           29.50%   
5          % American Indian & Alaska Native            0.50%   
6                                    % Asian            5.10%   
7   % Native Hawaiian/Other Pacific Islander            0.10%   
8                                 % Hispanic           10.10%   
9                       % Non-Hispanic White           52.60%   
10               % not proficient in English               3%   
11                                 % Females           51.50%   
12                                   % Rural            2.90%   
13                            Length of Life                    
14                           Life expectancy             76.2   
15          Premature age-adjusted mortality              440   
16                           Child mortality               80   
17                          Infant mortality                9   
18                           Quality of Life                    
19                Frequent physical distress              13%   
20                  Frequent mental distress              13%   
21                       Diabetes prevalence              12%   
22                            HIV prevalence              761   
23                          Health Behaviors                    
24                           Food insecurity              18%   
25           Limited access to healthy foods               9%   
26                      Drug overdose deaths               41   
27                Motor vehicle crash deaths               15   
28                        Insufficient sleep              37%   
29                             Clinical Care                    
..                                       ...              ...   
46                            Traffic volume              422   
47                             Homeownership              57%   
48                Severe housing cost burden              16%   
49                           Health Outcomes                    
50                            Length of Life                    
51                           Premature death            9,700   
52                           Quality of Life                    
53                       Poor or fair health              19%   
54                 Poor physical health days              4.3   
55                   Poor mental health days              4.3   
56                           Low birthweight              10%   
57                            Health Factors                    
58                          Health Behaviors                    
59                             Adult smoking              20%   
60                             Adult obesity              31%   
61                    Food environment index              6.6   
62                       Physical inactivity              26%   
63          Access to exercise opportunities              92%   
64                        Excessive drinking              19%   
65           Alcohol-impaired driving deaths              29%   
66           Sexually transmitted infections            738.3   
67                               Teen births               29   
68                             Clinical Care                    
69                                 Uninsured              13%   
70                   Primary care physicians          1,160:1   
71                                  Dentists          1,220:1   
72                   Mental health providers        510:01:00   
73                Preventable hospital stays            5,911   
74                     Mammography screening              41%   
75                          Flu vaccinations              43%   

   

**Table 4. Death by Age New York**<br>
Reference:https://www.worldometers.info/coronavirus/coronavirus-age-sex-demographics/

In [8]:
Death_by_Age_Table

AGE Number of Deaths Share of deaths  \
0   0 - 17 years old                9           0.06%   
1  18 - 44 years old              601           3.90%   
2  45 - 64 years old            3,413          22.40%   
3  65 - 74 years old            3,788          24.90%   
4      75+ years old            7,419          48.70%   
5              TOTAL           15,230            100%   

  With underlying conditions Without underlying conditions  \
0                          6                             3   
1                        476                            17   
2                      2,851                            72   
3                      2,801                             5   
4                      5,236                             2   
5               11,370 (75%)                     99 (0.7%)   

  Unknown if with underlying cond. Share of deaths of unknown + w/o cond.  
0                                0                                  0.02%  
1                              108                                  0.80%  
2                              490                                  3.70%  
3                              982                                  6.50%  
4                            2,181                                 14.30%  
5                    1,551 (24.7%)                                 25.30%

**Table 5. Capacity and Fatality Rates**<br>
Reference:https://bao.arcgis.com/covid-19/jhu/county/13051.html

In [3]:
Capacity_Table= pd.read_csv('/home/jovyan/demo/Hospital_Capacity.csv')
Capacity_Table 

Description  Duval Chatham  Queens
0     Number of Beds  3,456   1,569   3,210
1  Cases per 100,000    128     119   2,521
2    Confirmed Cases  1,215     344  57,459
3             Deaths     32      14   4,595
4      Fatality Rate   2.6%      4%      8%

## Discussion

Based on venues in Savannah(Table 1) and Jacksonville(Table 2) there are many businesses/venues that will be significantly impacted by economic shutdown. In North Jacksonville and East Jacksonville, fast food restaurants account for about 38% of venues. In East Savannah, pizza places(18%) and discount stores(18%) account for 36% of venues while in Southover, Savannah bakeries(25%), breweries(25%), and discount stores(25%) account for 75% of venues. With so many businesses impacted will reopening be risky? If we take a look at county health rankings(Table 3), only 14-15% of the population is 65 and over indicating a small high-risk population(Table 4) with about 75% of deaths occurring in people 65+. Additionally, access to exercise opportunities are about 90% indicating the ability for most of the population to engage in immune and health boosting activities which is important to improve the obesity rate at 31%. However, we do need to factor in smoking at 20% which increases the risk for complications for Coronavirus. Is there medical capacity for handling increase in cases? According to Johns Hopkins University, capacity and fatality rates(Table 5), Duval and Chatham counties fall well below the averages of Queens, New York. Although New York was initial site, at this point Jacksonville and Savannah have relatively low numbers of infections with a significant number of beds available. 



## Conclusion

The governors of Jacksonville and Savannah should consider the pros and cons of opening up too soon. As Table 5 indicates, the rate of infections and fatalities are relatively low especially for Duval county of Jacksonville. Chatham county has a higher rate but still half of Queens, New York which is the epicenter of the virus. With a population of about 300K, Chatham may fare better than other counties because of the relatively small population and vulnerable population at approximately 15%(65+). Governors have considered opening places at 25% capacity but many businesses can't be profitable at such low capacity. Another suggestion is perhaps sustain stay-at-home orders for vulnerable populations. Businesses like gyms may significantly lower the risks by permitting those under 65 at full capacity with significant cleaning measures. For schools, the most vulnerable populations continue to work online while the least can go back to the classroom. Allow the least vulnerable populations get back to work and subsidize the ones that need to stay at home.